pkg/version/base.go

```diff
@@ -51,9 +51,13 @@ var (
 	// semantic version is a git hash, but the version itself is no
 	// longer the direct output of "git describe", but a slight
 	// translation to be semver compliant.
+
+	// NOTE: The $Format strings are replaced during 'git archive' thanks to the
+	// companion .gitattributes file containing 'export-subst' in this same
+	// directory.  See also https://git-scm.com/docs/gitattributes
 	gitVersion   string = "v1.7.11-beta.0+$Format:%h$"
```

Comment:
this is gonna be stuck at a weird version forever?

Reply:
Yes I couldn't decide whether to leave the 'already on the branch' version intact or revert it.  Either way, it will get overwritten via the lib/version.sh change.

Reply:
maybe once we remove the "update files" logic from anago we can make a cleanup pass in the release branches to remove the bad versions here.

Reply:
I suppose.  I would have much rather fixed this if this was an issue on the first pass. :-/
How strongly do you feel about it?  I'm still on the fence about it.

Reply:
it's not a huge deal, but it might be a bit confusing if anyone actually looks at this file on the release branches.